<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordingForm</class>
 <widget class="QWidget" name="RecordingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Record</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_start_stop_record">
         <property name="text">
          <string>Start recording</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Name of the recrod:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEdit_recname">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_recorder_state">
         <property name="font">
          <font>
           <pointsize>35</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>123</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Replay</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_start_stop">
         <property name="title">
          <string>Replay:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_stop_replay">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton_start_replay">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_list">
         <property name="title">
          <string>Stored records:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_refresh">
            <property name="text">
             <string>Refresh</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QListWidget" name="listWidget_rectords"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_ctrl">
         <property name="title">
          <string>Replay controll:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Current time [min]:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3" colspan="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QPushButton" name="pushButton_pause">
            <property name="text">
             <string>Pause</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLabel" name="label_current_time">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_resume">
            <property name="text">
             <string>Replay</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
